entropylab/results_backend/sqlalchemy/tests/test_db.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import pytest
+
+from config import settings
+from entropylab import SqlAlchemyDB, RawResultData
+
+
+HDF_FILENAME = "./entropy.hdf5"
```

Comment:
This creates an `entropy.hdf5` in the root of the project when we run tests, and it is not ignored.

Can you create this file in some temp directory, and add that directory to ignore?